tests/test_integration.py

```diff
@@ -15,6 +16,10 @@ def test_integration(transactional_db, settings):
     connection.connect()
 
     # Assertions
-    assert str(connection.connection.server_version).startswith("12")
-    assert postgres_signals.get_hstore_oids("default") != hstore
-    assert postgres_signals.get_citext_oids("default") != citext
+    try:
+        assert str(connection.connection.server_version).startswith("12")
+        assert postgres_signals.get_hstore_oids("default") != hstore
+        assert postgres_signals.get_citext_oids("default") != citext
+    finally:
+        settings.DATABASES["default"]["PORT"] = defaultport
+        connection.connect()
```

Comment:
not fond of a try/ finally in tests. I'd much rather have a fixture even for a single test.